.changeset/stale-bags-eat.md

```diff
@@ -0,0 +1,15 @@
+---
+'braid-design-system': patch
```

Comment:
Feel like `minor` would be more appropriate. But won't matter either way.

Reply:
Changed.